src/bin/generate_and_fix_stubs.py

```
@@ -0,0 +1,250 @@
+        if e.stderr:
+            print(f"  Stderr: {e.stderr}")
+        return False
+    except FileNotFoundError:
+        print("✗ Could not find cargo command")
+        return False
+
+
+def process_pyi_files(directory: str = "../../oxapy") -> Dict[str, int]:
+    """Process all .pyi files in the directory."""
+    path = Path(directory)
+    fixer = StubFixer()
```

Ruff (0.13.2)

Local variable `fixer` is assigned to but never used

Remove assignment to unused variable `fixer`

(F841)